From 74680dfb3df4c0c5b0e4bcf41717a9ea16fd8680 Mon Sep 17 00:00:00 2001
From: Joe MacDonald <joe_macdonald@mentor.com>
Date: Mon, 29 Sep 2014 14:19:48 -0400
Subject: [PATCH] replcon: correct invalid prototype for lsetfilecon_raw

Port debian patch from:

        git://anonscm.debian.org/selinux/setools.git
        commit a3ab84b35efd9c42641d53ec2236ad01f7411df7

Upstream-Status: Denied [ the setools3 tree is in stasis and the focus is
                          only on setools4 now ]

Signed-off-by: Joe MacDonald <joe_macdonald@mentor.com>
---
 secmds/replcon.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/secmds/replcon.cc b/secmds/replcon.cc
index 34f7c1a..307c39f 100644
--- a/secmds/replcon.cc
+++ b/secmds/replcon.cc
@@ -60,7 +60,7 @@ static struct option const longopts[] = {
 	{NULL, 0, NULL, 0}
 };
 
-extern int lsetfilecon_raw(const char *, security_context_t) __attribute__ ((weak));
+extern int lsetfilecon_raw(const char *, const char *) __attribute__ ((weak));
 
 /**
  * As that setools must work with older libselinux versions that may
-- 
1.9.1

